test_pyjudge.py

```diff
@@ -0,0 +1,22 @@
+import unittest
```

Comment:
No need to add such heavy testing. We can use [pytest](https://docs.pytest.org/en/latest/) which will make tests a lot easier and cleaner to maintain.
```python
import server

def test_simple_math():
    assert 1 + 1 == 2
```

This will import the server file and verify that all code can be imported. Then we can run tests like `pytest test_pyjudge.py`